res/unix/Makefile

```diff
@@ -1,22 +1,42 @@
-POLARIS_BIN_DIR := ~/.local/bin/polaris
-POLARIS_DATA_DIR := ~/.local/share/polaris
+.PHONY: all build install install-bin install-data clean uninstall
+
+BINDIR ?= /usr/local/bin
+DATADIR ?= /usr/local/share
+LOCALSTATEDIR ?= /var
+RUNSTATEDIR ?= /run
+
+POLARIS_BIN_PATH := $(BINDIR)/polaris
+export POLARIS_WEB_DIR :=  $(DATADIR)/polaris/web
+export POLARIS_SWAGGER_DIR :=  $(DATADIR)/polaris/swagger
+export POLARIS_DB_DIR := $(LOCALSTATEDIR)/lib/polaris
+export POLARIS_LOG_DIR := $(LOCALSTATEDIR)/log/polaris
+export POLARIS_CACHE_DIR := $(LOCALSTATEDIR)/cache/polaris
+export POLARIS_PID_DIR := $(RUNSTATEDIR)/polaris
 
 all: build
 
 build:
 	cargo build --release
 
-install: build
-	install -d $(POLARIS_BIN_DIR)
-	install -d $(POLARIS_DATA_DIR)
-	install ./target/release/polaris $(POLARIS_BIN_DIR)
-	cp -r ./web $(POLARIS_DATA_DIR)
-	cp -r ./swagger $(POLARIS_DATA_DIR)
-	@echo "Polaris installation complete!"
+install: install-bin install-data
+
+install-bin: build
+	install -Dm755 ./target/release/polaris $(POLARIS_BIN_PATH)
+
+install-data:
+	install -d $(POLARIS_WEB_DIR)
+	install -d $(POLARIS_SWAGGER_DIR)
+	cp -r ./web $(POLARIS_WEB_DIR)
+	cp -r ./swagger $(POLARIS_SWAGGER_DIR)
 
 clean:
 	cargo clean
 
 uninstall:
-	rm -r $(POLARIS_BIN_DIR)
-	rm -r $(POLARIS_DATA_DIR)
+	rm $(POLARIS_BIN_PATH)
+	rm -r $(POLARIS_WEB_DIR)
+	rm -r $(POLARIS_SWAGGER_DIR)
+	rm -r $(POLARIS_DB_DIR)
+	rm -r $(POLARIS_LOG_DIR)
+	rm -r $(POLARIS_CACHE_DIR)
+	rm -r $(POLARIS_PID_DIR)
```

Comment:
I think data files shouldn't be removed on uninstall

Reply:
I can't find any official recommendation on this, but I think as a user I would prefer everything gone from my system. I will leave this as-is unless there is a strong argument for leaving files behind.